zombies.js

```diff
@@ -79,6 +96,59 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  let _pack = [];
+  let _maxHealth = health;
+
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+
+  this.getPack = function () {
+    return _pack;
+  }
+
+  this.getMaxHealth = function () {
+    return _maxHealth;
+  }
+}
+
+/** 
+* Player Class Method => isItInPack
+ * -----------------------------
+ * Player checks to see if the item is in the pack, 
+ * returns a boolean or index number depending on the return type.
+ * 
+ * @param {Item} item 
+ * @return {Number}
+ */
+
+Player.prototype.isItInPack = function (item) {
+  return this.getPack().indexOf(item);
+}
+
+/** 
+* Player Class Method => packRemoveOrReplace
+ * -----------------------------
+ * Player checks to see if the item is in the pack, 
+ * if it is removes the item or replaces it. Otherwise it 
+ * returns false
+ * 
+ * @param {Number} indexOfRemoval
+ * @param {Item} itemThatReplaces
+ */
+
+Player.prototype.packRemoveOrReplace = function (indexOfRemoval, itemThatReplaces) {
```

Comment:
As a private method, you might want to declare this in the constructor so outside code can't just replace things in your pack.